src/components/Pillbox/__snapshots__/Pillbox.test.js.snap

```diff
@@ -7,32 +7,57 @@ exports[`Pillbox renders 1`] = `
   <div
     data-stylename="styles.pill-container"
   >
-    <CSSTransitionGroup
-      transitionAppear={false}
-      transitionEnter={true}
-      transitionEnterTimeout={400}
-      transitionLeave={true}
-      transitionLeaveTimeout={300}
-      transitionName={
-        Object {
-          "enter": undefined,
-          "enterActive": undefined,
-          "leave": undefined,
-          "leaveActive": undefined,
-        }
-      }
+    <TransitionGroup
+      childFactory={[Function]}
+      component="div"
     >
-      <Pill
-        id={1}
+      <CSSTransition
+        classNames={
+          Object {
+            "enter": undefined,
```

Comment:
Is this how you disable some of the functionality of `CSSTransition`?

Reply:
im not sure why the snapshot shows undefined actually. something to do with how css modules work in snapshots?